spec/javascripts/components/accordion-spec.js

```diff
@@ -103,18 +103,21 @@ describe('An accordion component', function () {
       topLevelControl.click()
       expect(topLevelControl.innerText).toEqual('Hide all sections')
 
-      element.querySelectorAll('.gem-c-accordion__section').forEach(function (section) {
+      var sections = element.querySelectorAll('.gem-c-accordion__section')
+      for (var s = 0; s < sections.length; s++) {
+        var section = sections[s]
         expect(section).toHaveClass('gem-c-accordion__section--expanded')
         expect(section.querySelector('.gem-c-accordion__toggle-text').innerText).toEqual('Hide')
-      })
+      }
 
       topLevelControl.click()
       expect(topLevelControl.innerText).toEqual('Show all sections')
 
-      element.querySelectorAll('.gem-c-accordion__section').forEach(function (section) {
+      for (s = 0; s < sections.length; s++) {
```

Comment:
Instead of reassigning `s`, could this another variable?

```suggestion
      for (var x = 0; x < sections.length; x++) {
```

Reply:
It could, yes. What's the value of that? Wouldn't creating a new variable within this scope use (fractionally) more memory?

Reply:
Oh but this is within the test so not a big deal if it does, I guess.